l10n/sleha/fr-fr/xml/article_installation.xml

```diff
@@ -0,0 +1,748 @@
+   </para>
+
+  <sect2 xml:id="vl-ha-inst-quick-req-hw">
+   <title>Configuration matérielle requise</title>
+   <variablelist>
+    <varlistentry>
+     <term>Serveurs</term>
+     <listitem>
+      <para>
+       Deux serveurs exécutant les logiciels spécifiés à la <xref linkend="il-ha-inst-quick-req-sw"/>.
+      </para> <para>
+      Ces serveurs peuvent être des machines virtuelles ou sans système d&apos;exploitation. Ils ne nécessitent pas une configuration matérielle identique (mémoire, espace disque, etc.), mais doivent avoir la même architecture. Les grappes multi plate-forme ne sont pas prises en charge.
+     </para>
+      
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Canaux de communication</term>
+     <listitem>  <para>
+       Au moins deux médias de communication TCP/IP par noeud de grappe. L&apos;équipement réseau doit prendre en charge le moyen de communication que vous souhaitez utiliser pour les communications au sein de la grappe, à savoir la multidiffusion ou la monodiffusion. Les médias de communication doivent prendre en charge un débit de données de 100 Mbit/s ou plus. La prise en charge d&apos;une grappe requiert au moins deux chemins de communication redondants. Cette configuration peut être obtenue via :</para>
+       <itemizedlist>
+        <listitem>
+         <para>
+          Une liaison de périphérique réseau (de préférence).
+         </para>
+        </listitem>
+        <listitem>
+         <para>
+          Un deuxième canal de communication dans Corosync.
+         </para>
+        </listitem>
+       </itemizedlist> </listitem>
+    </varlistentry>
+    <varlistentry>
+     <term>Isolation de noeud/STONITH</term>
+     <listitem>  <para>
+      Pour éviter un scénario de <quote>vues de grappe divergentes</quote>, les grappes ont besoin d&apos;un mécanisme d&apos;isolation de noeud. Dans un scénario de vues de grappe divergentes, les noeuds de grappe sont divisés en deux groupes ou plus qui ne se connaissent pas (en raison d&apos;une défaillance matérielle ou logicielle, ou d&apos;une interruption de la connexion réseau). Un mécanisme d&apos;isolation isole le noeud en question (généralement en le réinitialisant ou en le mettant hors tension). Ce processus est également appelé STONITH (<quote>Shoot the other node in the head</quote>). Un mécanisme d&apos;isolation de noeud peut être un dispositif physique (un bouton marche/arrêt) ou un mécanisme de type SBD (STONITH par disque) associé à un système de surveillance (watchdog). L&apos;utilisation d&apos;un SBD nécessite un stockage partagé. 
+     </para> </listitem>
+    </varlistentry>
+   </variablelist>
+ </sect2>
+  <sect2 xml:id="il-ha-inst-quick-req-sw">
+   <title>CONFIGURATION LOGICIELLE REQUISE</title>
```

Comment:
Uppercase letters

Reply:
Fix now in XML on my local cloned repo and pushed to 'Vistatec/localisation checkin sleha15sp4' branch in repo.